src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -129,17 +128,20 @@ public static final class Drivetrain {
 
         //#region Command Constants
 
-        public static final double kSlowDriveSpeed = 0.25;
-        public static final double kSlowDriveRotation = 0.30;
+        public static final double kSlowDriveSpeed = 0.25; // Percent Multiplier
+        public static final double kSlowDriveRotation = 0.30; // Percent Multiplier
         public static final double kAlignMultiplier = 1D/3D;
         public static final double kAlignForward = 0.6;
 
-        public static final double chargeStationAlignTolerance = 2.5;
+        public static final double chargeStationAlignTolerance = 2.5; // Degrees
         public static final double chargeStationAlignSpeed = 0.3 / 13;
         public static final double chargeStationAlignTime = 500;
         public static final double chargeStationAlignFF = 0.1;
+        public static final double wheelTurnDriveSpeed = 0.0001; // Meters / Second ; A non-zero speed just used to orient the wheels to the correct angle. This should be very small to avoid actually moving the robot.
```

Comment:
The wheels can just spin in place. I don't think this should be needed, but I don't object.

Reply:
Agree that it's not needed, but `drive` already handles all the necessary kinematics/wheel optimization/etc. Better is the enemy of good enough (especially when we don't have to maintain 2 drive pipelines). In this case I think it should be fine. The code calls `dt.stop()` immediately after, so drive speed should (ultimately) get set to zero. This is just to get around the "don't move if speed == 0" line.